src/Voron/IndexEntryKeyGenerator.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Voron
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    public class IndexEntryKeyGeneratorAttribute : Attribute
```

Comment:
Wouldn't `src/Voron/Data/Tables` be more suitable location for that file?

Reply:
done